### PR TITLE
Add QFN-20-1EP_5x5mm_P0.65mm_EP3.1x3.1mm

### PR DESCRIPTION
**Pull request by chschlue**
https://pdfserv.maximintegrated.com/land_patterns/90-0009.PDF
https://pdfserv.maximintegrated.com/package_dwgs/21-0140.PDF

### CODE REVIEW

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-20.yaml [hunk 1]
```diff
@@ -542,6 +542,69 @@ QFN-20-1EP_4x5mm_P0.5mm_EP2.65x3.65mm:
...
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://pdfserv.maximintegrated.com/package_dwgs/21-0140.PDF'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 5
+    tolerance: 0
```
Comment (evanshultz) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-20.yaml]: I believe you can remove this line and just leave `nominal`.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-20.yaml [hunk 2]
```diff
@@ -542,6 +542,69 @@ QFN-20-1EP_4x5mm_P0.5mm_EP2.65x3.65mm:
...
+  lead_len:
+    minimum: 0.45
+    nominal: 0.55
+    maximum: 0.65
+
+  EP_size_x:
+    minimum: 3.0
+    nominal: 3.1
+    maximum: 3.2
+  EP_size_x_overwrite: 3.1
```
Comment (evanshultz) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-20.yaml]: Remove this as the nominal is the EP size you want. Same with Y dimension below.

**Comment by evanshultz** [comment]
Also call this TQFN and not just QFN. The 3D models will be different.

**Comment by chschlue** [comment]
Done

**Comment by evanshultz** [comment]
@pointhi @poeschlr 
Footprint has been merged.

**Pull request merged**